README.md

```diff
@@ -112,8 +110,7 @@ end
 
 Your answer:
 ```text
-
-Replace this with your answer
+Context and describe do the exact same thing. The only reason there are two words to describe the exact same thing is for readability. Describe can be used to talk about specific objects or methods, and context can be used to describe the context.
```

Comment:
:+1:

I can hear my 7th grade English teacher now, "Don't use the word to define itself." :)